client/src/models/Label.js

```diff
@@ -80,4 +80,16 @@ export default class extends BaseModel {
       default:
     }
   }
+
+  static findLabelsFromText(filterText, labels) {
+    const selectLabel = filterText.toLocaleLowerCase();
+    const matchingLabels = labels.filter((label) =>
+      label.name.toLocaleLowerCase().startsWith(selectLabel),
```

Comment:
Label name can be empty, and if there is such a label, an error `Cannot read properties of null (reading 'toLocaleLowerCase')` will occur.

Reply:
I'll fix this